src/features/contextPanel/index.tsx

```diff
@@ -14,31 +14,28 @@ interface ContextPanelProps {
   containers: NamedContainers;
   position?: BCVWP;
   visibleSourceVerses: Verse[];
+  sx?: Record<string, unknown>;
```

Comment:
This should be renamed to something other than `sx` like `style` since `sx` is used for the MUI components which this prop doesn't apply to. Also, this should be of type `Partial<React.CSSProperties`